### DISCUSSION TITLE
Release notes doesn't work for polish language

### DISCUSSION THREAD

**Discussion by The-Redhat** [codemagic-ci-cd/codemagic-docs#2638] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Steps to reproduce
i create the release notes like
```
[
  {
    "language": "pl",
    "text": "minor bug fixes"
  }
]
```

### Expected results

auto fill `What's new` in app store

### Actual results

not filling out for the polish language

**Comment by dtrdic** [comment, marked as answer]
@The-Redhat should work now, please try again
